addons/oci-private-dns/README.md

```diff
@@ -0,0 +1,160 @@
+- Prod VCN resolver evaluates the query based on the [VCN DNS Resolver processing order](#VCN-DNS-Resolver-processing-order) as follows:<br>
+   ⓵ **Associated Private Views** - Since no Private view is associated with the Prod VCN resolver, it proceeds to the next step.<br>
+   ⓶ **Default Private View** - The default private view contains a DNS record for the database, so the resolver retrieves the record.
+- The Prod VCN resolver returns the DNS response to **web01-p**.
+
+<img src="https://github.com/oracle-quickstart/terraform-oci-open-lz/blob/content/addons/oci-private-dns/withinspoke.gif" width="800" />
+
+&nbsp;
+
+#### Scenario 2: Spoke to Spoke DNS resolution
+- The **web01-p.ssnpweb.vcnprod.oraclevcn.com** in prod Spoke VCN performs *nslookup* to retrieve the IP address of **web02-pp.ssnppweb.vcnpreprod.oraclevcn.com**, located in the preprod Spoke VCN.
+- Prod VCN resolver evaluates the query based on the [VCN DNS Resolver processing order](#VCN-DNS-Resolver-processing-order) as follows:<br>
+   ⓵ **Associated Private Views** - Since no Private view is associated with the Prod VCN resolver, it moves to the next option.<br>
+   ⓶ **Default Private View** – This view contains only DNS records specific to the Prod VCN. Since the resolver does not have a record for **web02-pp**, it proceeds to the next step.<br>
+   ⓷ **Forwarding Rules** – The resolver identifies a forwarding rule for **oraclevcn.com**, directing the query to the **hub_dns_listener** through the **p_dns_forwarder** in the Prod Spoke VCN.
+- The Hub VCN Resolver has all DNS data/records from the **Associated Private views**, processes the query and returns the DNS response to **web01-p** in the Prod Spoke VCN.
+
+<img src="https://github.com/oracle-quickstart/terraform-oci-open-lz/blob/content/addons/oci-private-dns/spoke2spoke.gif" width="800" />
+
+&nbsp;
+
+## **2. Multi-Region: Private DNS configuration view**
+Configuration details:
+  - Each Hub VCN in a given region consists of the following resources and components:
+    - Forwarding (**hub_dns_forwarder-1 and 2**) and Listening (**hub_dns_listener-1 and 2**) endpoints.
+    - The Hub VCN Resolver is associated with Private views for both - the Hub and Spoke VCNs, allowing it to store and resolve all DNS records across the three VCNs within the Hub and Spoke architecture.
+    - Conditional Forwarding Rules in the Hub VCN resolvers ensure that region-specific DNS zones are forwarded to the appropriate region.
+  - In each region, Spoke VCN Resolvers include:
+    - Forwarding Endpoints: p_dns_forwarder-1 and p_dns_forwarder-2.
+    - Conditional forwarding rules that direct queries for **oraclevcn.com**, **oraclecloud.com** and **oci.customer-oci.com** domains to the respective  **hub_dns_listener-1 or 2** for resolution.
+  
+&nbsp;
+<img src="images/multi-region.png" width="900" height="value">
+
+&nbsp;
+
+&nbsp;
+
+## Multi-Region: Private DNS animation
```

Comment:
I'd maybe rather call the section Multi-Region: Private DNS Flows or (Scenarios)

Is this part of the 2. Section if sou maybe make the title smaller by ### so it's clear, possibly numbering for sub-sections as well?